modules/core/ui/theme/farm_ui_theme.module

```diff
@@ -22,6 +22,13 @@ function farm_ui_theme_theme($existing, $type, $theme, $path) {
   ];
 }
 
+/**
+ * Implements hook_theme_registry_alter().
+ */
+function farm_ui_theme_theme_registry_alter(&$theme_registry) {
+  $theme_registry['comment']['path'] = \Drupal::service('extension.list.module')->getPath('farm_ui_theme') . '/templates';
+}
+
```

Comment:
We should use a theme suggestion for comment templates instead of overriding `comment.html.twig` eg: `comment--asset`, `comment--log`, `comment--asset--land`. I would hope these suggestions exist but if not we could add them

Reply:
Wow.... there's only a suggestion for the referenced entity type's bundle. Only thinking about nodes, I see :disappointed: 

![Screenshot from 2023-04-14 13-41-52](https://user-images.githubusercontent.com/3116995/232150975-dae925d5-548e-4566-83af-95ff284caab7.png)

Reply:
If we can avoid the heavy handed approach that would be great, template suggestions are kinda designed for this. Instead of a template for each entity type I think we could provide a single generic `comment--farm` template and add this as a template suggestion for asset/log/etc comment types that would like to use the farmOS convention. Should also be possible for templates to inherit from one another, so having multiple templates may not be a big deal (no duplication).

Reply:
Happy to investigate this and give it a try!

Reply:
That makes sense! Please do! Feel free to push commit(s) to this branch.